### PR TITLE
markdown space fix

### PR DESCRIPTION
**Pull request by victazzz**

### CODE REVIEW

File: src/utils/components/markdown/index.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    )
+    .replace(
+      getRegexBetween(":{3}tip", ":{3}", "gs"),
+      '<div className="notice-block notice-block--tip">$6</div>'
+    );
+};
+
+export const fixContentForMarkdown = (content = ""): string => {
+  if (content === null || content === "null") return "";
+  return flow([
```
Comment (pieecia) [inline src/utils/components/markdown/index.ts:68]: // WARNING: This is not a drop in replacement solution and
// it might not work for some edge cases. Test your code! 
```
const flow =
  funcs =>
  (...args) =>
    funcs.reduce((prev, fnc) => [fnc(...prev)], args)[0]
```
    
@victazzz moze bez loadasha bedzie ok? https://youmightnotneed.com/lodash#flow
